solidity/ecdsa/test/WalletRegistry.WalletCreation.test.ts

```diff
@@ -1077,194 +1162,207 @@ describe("WalletRegistry - Wallet Creation", async () => {
               })
             })
 
-            context("with invalid dkg result submitted", async () => {
```

Comment:
The changes below are a result of adding "...and challenged" to the context description and formatting the code with new indentation.